client/lib/reader-lists-tags/store.js

```diff
@@ -0,0 +1,90 @@
+// Reader Lists Tag Subscription Store
+
+// External dependencies
+import { List, Map, fromJS } from 'immutable';
+import debugModule from 'debug';
+
+// Internal dependencies
+import { action as actionTypes } from 'lib/reader-lists-tags/constants';
+import { createReducerStore } from 'lib/store';
+
+const debug = debugModule( 'calypso:reader-lists-tags' ); //eslint-disable-line no-unused-vars
+
+const initialState = {
+	tags: Map(), // eslint-disable-line new-cap
+	errors: [],
+	currentPage: {},
+	isLastPage: {},
+	isFetching: false
+};
+
+function getListTags( state, listId ) {
+	return state.get( 'tags' ).get( parseInt( listId ), List() ); // eslint-disable-line new-cap
```

Comment:
Does this make a new list instance on every call? Might want to stash the default somewhere and reuse it.